pkg/plugin/v3/scaffolds/init.go

```diff
@@ -57,6 +56,7 @@ type initScaffolder struct {
 
 // NewInitScaffolder returns a new Scaffolder for project initialization operations
 func NewInitScaffolder(config *config.Config, license, owner string) scaffold.Scaffolder {
+
```

Comment:
the changes made here are only about layout and do not really change its behaviour. IMO would better keep only what is required to introduce the feat.